media_cdn/snippets.py

```diff
@@ -74,7 +82,7 @@ def sign_url(url: str, key_name: str, base64_key: str, expiration_time: datetime
     return signed_url
 
 
-def sign_url_prefix(url: str, url_prefix, key_name: str, base64_key: str, expiration_time: datetime.datetime) -> str:
+def sign_url_prefix(url: str, url_prefix: str, key_name: str, base64_key: str, expiration_time: datetime.datetime) -> str:
```

Comment:
beware of line length.
```suggestion
def sign_url_prefix(
    url: str,
    url_prefix: str,
    key_name: str,
    base64_key: str,
    expiration_time: datetime.datetime,
) -> str:
```